roles/provision-metrics-apb/tasks/provision-app-metrics.yml

```diff
@@ -0,0 +1,115 @@
+- name: Postgresql PVC
+  k8s_v1_persistent_volume_claim:
+    name: postgresql-data
+    namespace: "{{ namespace }}"
+    labels:
+      app: aerogear-app-metrics
```

Comment:
@david-martin I used aerogear-app-metrics as that's the name we're going with, but is it a good idea to drop the aerogear here for productisation later?

Reply:
I wouldn't worry about it.
There are likely more changes than just the image name, so not worth considering at this time

Reply:
:+1: